Gemfile

```diff
@@ -43,7 +43,10 @@ gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 gem "bootsnap", require: false
 
 # Use Sass to process CSS
-# gem "sassc-rails"
+gem "sassc-rails"
+
+# Use gem Devise to configurate authorization and authentication to Users
+gem 'devise'
```

Comment:
Devise only handles authentication (question of "who is this user")

Authorization can be handled with the [pundit](https://github.com/varvet/pundit) gem (question of "is this user allowed to do X")